src/test/java/com/savvato/tribeapp/services/ConnectTestConstants.java

```diff
@@ -25,7 +26,7 @@
 import static org.mockito.Mockito.*;
 
 @ExtendWith({SpringExtension.class})
-public class ConnectServiceImplTest extends AbstractServiceImplTest {
+public class ConnectTestConstants extends AbstractTestConstants {
```

Comment:
This should still be named ConnectServiceImplTest. 

`public class ConnectServiceImplTest extends AbstractTestConstants`

Reply:
The same principle applies to all the other changes, too.

Reply:
I missed this change. Thank you for catching that. I saw that the extends class was changed, but I don't know why it automatically adjusted the names of the classes.  I'll address that.